.circleci/config.yml

```diff
@@ -102,6 +102,8 @@ jobs:
             cd packages/graphql
             pub global activate coverage
             pub get
+            # warm-up dart2js, this is a dirty workaround
+            dart2js test/websocket_test.dart
```

Comment:
Could you explain why this ‘dirty workaround‘ is necessary?

Reply:
IDK, but it might not be. I added it back when CI started failing because I thought it might have been the cause.

Reply:
in the last ~10 CI running, the browser test always fails to compile the first file 
```
00:04 +0 -1: compiling test/multipart_upload_test.dart [E]                                                                                                                                             
  Failed to load "test/multipart_upload_test.dart": dart2js failed.
```
and the failing file is also random. I'm trying to figure out, but not yet succeed. If I can't figure in time, I suggest we temporarily disable `pub run test test -p chromium`